release_notes/Service_Fabric_ReleaseNotes_81.md

```diff
@@ -25,12 +25,12 @@ Microsoft Azure Service Fabric 8.1 Release Notes
 
 
 ## Key Announcements
-* Add support for Auxiliary Replica
-* **Preview** Add support for .NET 6.0 Service Fabric applications
-* Add API support application description update
-* Add periodic ping between RA and RAP to detect IPC failure and process stuck
-* Add support Probes for non containerized applications
-* Make cluster upgrade for node capacity updates impactless
+* Added support for Auxiliary Replica
+* **(Preview)** Added support for .NET 6.0 Service Fabric applications
+* Added API support for updating application descriptions
+* Added periodic ping between Reconfiguration Agent (RA) and Reconfiguration Agent Proxy (RAP) to detect IPC failure and process stuck
+* Added support probes for non containerized applications
```

Comment:
Align this with the "desc" below? "Added support for liveness and readiness probes for non containerized applications"

Reply:
done